### ISSUE TITLE
Alternative Name?

### ISSUE THREAD

**Issue by eonist** [sentryco/UpgradeAlert#1]
Zynch, Nexo, Zalar, UpApp, AppBoost

**Comment by adityasaravana** [comment]
Personally think that UpgradeAlert is very clear and makes the most sense. If you really do want to change the name, UpdateAlert would work, too.

**Comment by eonist** [comment]
yes. UpdateAlert is a viable alternative. I usually make these comments about possible alternatives so I can revisit them later if a competitor dups the name, or if apple decides to make the name an internal library. Or sometimes even if I want to explore making the open source project into something bigger. Like a "Launch darkly" service of sorts etc. Thanks for your input 🙏
